packages/xstate-wallet/src/channel-wallet.ts

```diff
@@ -273,9 +273,11 @@ export class ChannelWallet {
             break;
           case 'deposit':
             if (this.store.depositsSubmitted[channelId]) {
-              throw new Error(
-                `Attempting to submit a deposit for a channel with already submitted deposit ${this.store.depositsSubmitted[channelId]}`
+              logger.warn(
+                'Attempting to submit a deposit for a channel with already submitted deposit',
+                {depositSubmitted: this.store.depositsSubmitted[channelId]}
               );
+              continue;
```

Comment:
Why is this change in 69c942e9c61e06a950f973318ce77c0e49094aa4? Is it from prettier formatting on save?

If yes, this seems to indicate that `xstate-wallet` is not "up to code" WRT tooling -- we should lint on commit, as we do in other packages.

Reply:
That commit includes too much code.

This is a logic change to the behavior of the browser wallet. This is a better way to track submitted deposits: 33b2f3720

Reply:
ah, I did not read the diff carefully, and assumed it was generated by prettier.

Are the arguments in the correct order? In a vanilla pino logger, the message would come second, right?

Reply:
Good catch on the arguments. That line has been ripped out in 33b2f37 though